### PR TITLE
Warehouses API

### PR DESCRIPTION
**Pull request by bluemanos**

### CODE REVIEW

**Comment by viezel** [comment]
@bluemanos Please resolve conflicts

File: src/Providers/RepositoryServiceProvider.php [hunk 1]
```diff
@@ -39,6 +39,8 @@
 use Laracommerce\Core\Provinces\Repositories\Interfaces\ProvinceRepositoryInterface;
 use Laracommerce\Core\Provinces\Repositories\ProvinceRepository;
 use Illuminate\Support\ServiceProvider;
+use Laracommerce\Core\Warehouse\Repositories\Interfaces\WarehouseRepositoryInterface;
+use Laracommerce\Core\Warehouse\Repositories\WarehouseRepository;
 
 class RepositoryServiceProvider extends ServiceProvider
```
Comment (viezel) [inline src/Providers/RepositoryServiceProvider.php]: This file has been removed. please add binding in the LaracomCoreServiceProvider instead.

**Comment by bluemanos** [comment]
@viezel done :)

**Comment by viezel** [comment]
The diff coverage is 33.33%. . WOW. can you look into that?

**Pull request merged**